deployment/ccip/changeset/v1_6/cs_add_new_chain_e2e_test.go

```diff
@@ -424,6 +424,13 @@ func TestAddAndPromoteCandidatesForNewChain(t *testing.T) {
 				// RMNRemoteConfig:   &v1_6.RMNRemoteConfig{...}, // TODO: Enable?
 			}
 
+			// deploy donIDClaimer
```

Comment:
Can we add a test to confirm expected behavior when running AddCandidatesForNewChain?